src/Compiler/Checking/NicePrint.fs

```diff
@@ -1804,13 +1804,33 @@ module TastDefinitionPrinting =
             | Some gRef, Some sRef -> isPublicAccess gRef.Accessibility && isPublicAccess sRef.Accessibility
             | _ -> false
 
+        let (|MixedAccessibilityGetterAndSetter|_|) (pinfo: PropInfo) =
+            if not (pinfo.HasGetter && pinfo.HasSetter) then
```

Comment:
Probably could be done as one match expression, for more clarity?

Reply:
I wasn't sure how safe this would be if you look at 
https://github.com/dotnet/fsharp/blob/fd2571985589d90c0fd3abe55d3a9946f5df9fcf/src/Compiler/Checking/infos.fs#L1984-L1993

Reply:
Hmmm alright, well, this is not very important anyway so nevermind.